src/EditorFeatures/CSharpTest/AddUsing/AddUsingTests_NuGet.cs

```diff
@@ -268,6 +268,6 @@ class C
         private Task<ImmutableArray<PackageWithTypeResult>> CreateSearchResult(params PackageWithTypeResult[] results)
             => Task.FromResult(ImmutableArray.Create(results));
 
-        private IReadOnlyList<string> CreateNameParts(params string[] parts) => parts;
+        private ImmutableArray<string> CreateNameParts(params string[] parts) => parts.ToImmutableArray();
```

Comment:
:bulb: At this point you might as well just replace calls to this method with `ImmutableArray.Create(...)`.